tests/xfstests/install.pm

```diff
@@ -45,6 +45,10 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref', timeout => 600);
     zypper_call('in qa_test_xfstests',        timeout => 1200);
 
+    # Install upstream fio
+    script_run('git clone git://git.kernel.dk/fio.git');
+    script_run('cd fio && ./configure && make && make install && cd ..');
```

Comment:
Is really necessary to compile test tool in openqa?

Reply:
IMO this not cost too much, in my virtual machine it finished in 10 seconds. Some sub tests need fio to run test, and we don't have fio in product. It's stable in that git repo. I don't see big risk to compile.

Reply:
Whats wrong with using package from openSUSE or from QA repositories?